clib/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "clib"
```

Comment:
no prefix?

Reply:
I was on the fence since I like the shortness of `clib::` and mirroring `librad`. But I'm happy to do `radicle-clib` or `clibrad` :smile:

Reply:
As I said elsewhere, I think we should just prefix every crate with `link-`, and never need to change it again